android/src/main/java/org/ergoplatform/android/ergoauth/ErgoAuthenticationFragment.kt

```diff
@@ -80,7 +81,7 @@ class ErgoAuthenticationFragment : AbstractAuthenticationFragment(), WalletChoos
     }
 
     override fun startAuthFlow() {
-        if (authenticationWalletConfig?.secretStorage == null) {
+        if (authenticationWalletConfig?.isReadOnly() != false) {
```

Comment:
Unfortunately it can not as the expression output is `Boolean?` not `Boolean`. [See here](https://stackoverflow.com/a/32848624/7487013)

Reply:
How about `if (authenticationWalletConfig?.isReadOnly() ?: true)`

Reply:
Matter of taste, we can change to that one

Reply:
I began to make the change, but Linter suggests to avoid the `?:` operator and Alt-Enter reverts back to the code actually used. So I would suggest to leave it as it is?